writing_your_start-up_command.md

```diff
@@ -80,20 +80,28 @@ Explain. Also explain you can assign an alias to the script itself. FIXME
 
 Each directory within music will show up as a crate inside xwax, etc. FIXME
 
+(Fixed so it can take folders names with spaces and some illegal characters.)
```

Comment:
ur right, first time i do a pull request didnt know i could leave comments of the changes afterwards. :D